packages/nextjs/components/Header.tsx

```diff
@@ -94,7 +94,9 @@ export const Header = () => {
         </div>
         <Link href="/" passHref className="hidden lg:flex items-center gap-2 ml-4 mr-6 shrink-0">
           <div className="flex relative w-10 h-10">
-            <Image alt="SE2 logo" className="cursor-pointer" fill src="/logo.svg" />
+            <Image src="/batch15-logo.png" alt="BuidlGuidl Batch 15 Logo" width={300} height={300} />
+
+            {/* <Image alt="SE2 logo" className="cursor-pointer" fill src="/logo.svg" /> */}
```

Comment:
It's bad practice to leave commented code. Other contributors might be confused about whether it should be uncommented or removed.